src/aws-cur/cost-report.ts

```diff
@@ -150,6 +174,8 @@ export class CostReport extends Construct {
       additionalSchemaElements: ['RESOURCES'],
     });
     reportDefinition.node.addDependency(this.reportBucket.policy!);
+
+    this.costReportName = reportDefinition.ref;
```

Comment:
To obtain an auto-generated name.

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-cur-reportdefinition.html#aws-resource-cur-reportdefinition-return-values